### PR TITLE
Add link to "Development setup" from root

### PR DESCRIPTION
**Pull request by deedeeen**
The README of `app-web`, _Development setup_ does not yet have a link to the project root.
This request serves more as a quality-of-life improvement.

### CODE REVIEW

File: packages/app-web/README.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ Core web app built on React and Redux Toolkit
 
 ## Development setup
 
-Follow the instructions in the 'Development setup' section in the project root.
+Follow [the instructions](https://github.com/wierkstudio/ciphereditor#development-setup) in the 'Development setup' section in the project root.
```
Comment (ffraenz) [inline packages/app-web/README.md:8]: We should use relative paths here, as the repository may be viewed outside of GitHub.

```suggestion
Follow [the instructions](../../README.md#development-setup) in the 'Development setup' section in the project root.
```

**Review by ffraenz — CHANGES_REQUESTED** [review]
I see what you did there 👀
I've left some comments we should address before mering.

File: packages/app-web/README.md (hunk 1 above)
Comment (deedeeen) [reply packages/app-web/README.md]: Very fair - I'm currently somewhat inexperienced right now, so this should be good. :)

**Review by ffraenz — APPROVED** [review]
Looks perfect!

**Pull request merged**